### PR TITLE
Move feature flags to github hosted TOML config

### PR DESCRIPTION
**Pull request by chidg**

### CODE REVIEW

File: apps/extension/src/core/handlers/Extension.ts [hunk 1]
```diff
@@ -97,12 +101,43 @@ export default class Extension extends ExtensionHandler {
       DEBUG || TEST ? 0 : 2000
     )
 
+    // setup polling for config from github
+    let CONFIG_UPDATE_INTERVAL = 1000 * 60 * 5 // 5 minutes
+    const MAX_CONFIG_UPDATE_INTERVAL = 1000 * 60 * 60 // 1 hour
+    setInterval(() => {
+      try {
+        this.getRemoteConfig()
```
Comment (0xKheops) [inline apps/extension/src/core/handlers/Extension.ts]: async function here, errors woudln't be caught.
might want to use prom.catch() pattern

File: apps/extension/src/core/handlers/Extension.ts [hunk 2]
```diff
@@ -97,12 +101,43 @@ export default class Extension extends ExtensionHandler {
...
+      } catch (e) {
+        // exponential backoff for rate limits
+        if (
+          (e as Error).message === CONFIG_RATE_LIMIT_ERROR &&
+          CONFIG_UPDATE_INTERVAL < MAX_CONFIG_UPDATE_INTERVAL
+        )
+          CONFIG_UPDATE_INTERVAL = CONFIG_UPDATE_INTERVAL / 2
```
Comment (0xKheops) [inline apps/extension/src/core/handlers/Extension.ts]: if exponential fallback i guess timeout must be increased

```suggestion
          CONFIG_UPDATE_INTERVAL = CONFIG_UPDATE_INTERVAL * 2
```

File: apps/extension/src/core/handlers/Extension.ts [hunk 3]
```diff
@@ -97,12 +101,43 @@ export default class Extension extends ExtensionHandler {
...
     this.initDb()
     this.initWalletFunding()
     this.checkSpiritKeyOwnership()
     this.cleanup()
   }
 
+  private async getRemoteConfig() {
+    const config = await getConfig()
+
+    featuresStore.set({
```
Comment (0xKheops) [inline apps/extension/src/core/handlers/Extension.ts]:
```suggestion
    await featuresStore.set({
```

**Review by 0xKheops — CHANGES_REQUESTED** [review]
found a problem on error & exponential fallback logic

also, i thought we were using onfinality api key for substrate still, but our logic prevented that as it was using the flag suffixed with _EVM.
I'm wondering if we should add it back by setting apikey on chaindataProvider object

File: apps/extension/src/core/handlers/Extension.ts (hunk 3 above)
Comment (0xKheops) [reply apps/extension/src/core/handlers/Extension.ts]: if method doesn't return anything, might want to rename it to `updateRemoteConfig`

Comment (chidg) [reply apps/extension/src/core/handlers/Extension.ts]: renamed to `fetchRemoteConfig`

**Review by 0xKheops — APPROVED** [review]
LGTM

**Pull request merged**